modules/basegl/src/processors/volumeprocessing/volumeglprocessor.cpp

```diff
@@ -68,13 +69,14 @@ FrameBufferObject& getActiveFBO(Texture3D& texture, std::span<VolumeGLProcessor:
         it = std::ranges::min_element(
             fbos, [](const auto& a, const auto& b) { return a.lastUsed < b.lastUsed; });
         it->fbo.activate();
-        it->fbo.attachColorTexture(&texture, 0);
+        it->drawBuffers[0] = it->fbo.attachColorTexture(&texture, 0);
         it->textureId = texture.getID();
     } else {
         it->fbo.activate();
     }
     it->lastUsed = std::chrono::high_resolution_clock::now();
 
+    glDrawBuffers(static_cast<GLsizei>(it->drawBuffers.size()), it->drawBuffers.data());
```

Comment:
No need for drawbuffer array when using a single attachment.
```suggestion
    // change in class declaration.
    GLenum drawBuffer;
    
     it->drawBuffer = GL_COLOR_ATTACHMENT0;
    
    glDrawBuffer(it->drawBuffer);
```

Reply:
was considering adding more... but went a different way.